web_console/libs/http.js

```diff
@@ -5,4 +5,10 @@ export const client = ky.create({
   throwHttpErrors: false,
 });
 
-export const fetcher = (url, options = {}) => client.get(url, options).json();
+export const fetcher = (url, options = {}) => {
+  options.headers = {
+    ...options.headers,
+    'X-Federation-Id': localStorage.getItem('federationID')
```

Comment:
header is string value, so null value would be converted to `"null"`, `X-Federation-Id` could only be number. ways to fix this:
- don't set `X-Federation-Id` header for non-numeric value (**recommended**)
- convert non-numeric value to `""`